File: program/convert-seconds-to-hours/ConvertSecondsToHours.kt
import java.util.Scanner

fun main() {
    print("Enter Seconds : ")
    println()
    var sec = readLine()

    println("$sec seconds in hours : " + ConvertSecondsToHours(sec!!.toInt()))

}

private fun ConvertSecondsToHours(sec: Int): Int {
    val hour = sec / 3600;

    return  hour
}

File: program/program/convert-seconds-to-hours/ConvertSecondsToHours.cs
using System;
class MainClass {
    public static void Main (string[] args) {
    Console.WriteLine ("Please enter the seconds : ");
    double seconds = Convert.ToDouble(Console.ReadLine());
    double hours = seconds / 3600;
    Console.WriteLine(seconds+" seconds is "+hours+" hours");
  }
}
